### ISSUE TITLE
Change text color for light theme

### ISSUE THREAD

**Issue by bantuist** [panz3r/hyper-spotify#16]
Hey! How would I change the text color to be visible with a light theme?

**Comment by panz3r** [comment]
Hi @bantuist,

Thanks for your feedback, I'll add this request as a feature in the next release. 

For now you can try to juggle with CSS inside your `~/.hyper.js` as follow:
```js
...
    // custom css to embed in the terminal window
    termCSS: `
      footer > div > div > div {
        background-color: #1ED760 !important;
      }

      footer > div > div span,
      footer > div > div b {
        color: #1ED760 !important;
      }
    `,
...
```
This will make the buttons and text (except for the Spotify icon, visible when Spotify is not running) appear `Spotify Green` (`#1ED760`)

**Comment by panz3r** [comment]
Available in `hyper-spotify` version `3.1.0-beta.2`

If you'd like to give it a try, you should be able to install it by adding the version `#3.1.0-beta.2` inside `~/.hyper.js` like this
```js
module.exports = {
  ...
  plugins: ['hyper-spotify#3.1.0-beta.2']
  ...
}
```
